lib/Config.php

```diff
@@ -132,6 +132,14 @@ public function canUserEnableSIP(IUser $user): bool {
 		return $this->canEnableSIP[$user->getUID()];
 	}
 
+	public function isRecordingEnabled(): bool {
+		$isSignalingInternal = $this->getSignalingMode() === self::SIGNALING_INTERNAL;
+		$callRecordingCapability = $this->config->getAppValue('spreed', 'call_recording', 'no');
```

Comment:
Please document in docs/settings.md but also the default should be yes, we don't want people to first have to enable this